### PR TITLE
Feature/1849/windowtitle

### LINKED ISSUE #1849: add branding.name to window title
**Is your feature request related to a problem? Please describe.**

I would like to be able to use this application with my password manager that uses window detection based on the window title. Filebrowser does not have a unique windowtitle (Login is the default) and therefore it does not detect the correct entry.

**Describe the solution you'd like**
The most straigthforward option would be to add branding.name to the windowtitle if set. So the default would still be 'Login', but if branding was enabled it would be 'Login - branding.name'. It would also be an option to always make the windowtitle 'Login - Filebrowser' and only change Filebrowser to the branding if set.

### PR DESCRIPTION
**Pull request by newhinton**
**Description**
adds branding to document.title for branding/passwordmanager-use
closes #1849

### CODE REVIEW

File: frontend/src/views/Settings.vue [hunk 1]
```diff
@@ -61,6 +62,24 @@ export default {
...
+      if (value === true) {
+        this.fetchData();
+      }
+    },
+  },
+  methods: {
+    ...mapMutations(["setLoading"]),
+    async fetchData() {
+      document.title = document.title + ` - ${this.name}`;
+    },
```
Comment (o1egl) [inline frontend/src/views/Settings.vue]: Why is this added?

Comment (newhinton) [reply frontend/src/views/Settings.vue]: For consistentcy

it is not required to solve my linked issue, but without it we have different titles:

Login - Filebrowser
Files - /Path
Settings

with it we always append the title


Or do you mean the watch-function etc?

Comment (o1egl) [reply frontend/src/views/Settings.vue]: The watch function

Comment (newhinton) [reply frontend/src/views/Settings.vue]: Switching tabs in the settings also updates the title. If the watch-function is omitted, the instance-name only gets added to the initial load of the settings, but switching tabs removes it again

**Comment by ramiresviana** [comment]
Instead of manipulating the document title on every view, a better approach would be setting the title correctly on the first place. This can be achieved by changing this line:

https://github.com/filebrowser/filebrowser/blob/b2ad3f73686a2abaa4fc62963fba6f83c9da9b5e/frontend/src/router/index.js#L143

**Comment by newhinton** [comment]
@ramiresviana That does not seem to work properly. While you can briefly see the changed name (using `document.title = to.name + "TEST";`) it gets overriden as soon as the page is finished loading. 

If there is a better way to do this i am all ears, but this seemed the most straitforward way to me, especially for the file-view

Edit: it seems to be working partially, but not for the file-view because that overrides its title depending on the path. I will investigate further.

**Comment by newhinton** [comment]
@ramiresviana Thanks for your suggestion!
